app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,44 @@
+class PostsController < ApplicationController
+  def index
+    first_post = Post.first
+    if first_post
```

Comment:
I'm not sure I understand why this `index` action does a redirect in every path.
Shouldn't an index action return a list of items?

Reply:
This is because the infinite pagination will be done using the `show` action, maybe we can change this to use always the index and not only just one post, instead using a collection of them, at least 3 for example

Reply:
we can discuss this on a call @andresg4 , thanks for the contribution 🚀